tests/PySys/tedge/call_tedge_-h_-V/run.py

```diff
@@ -29,5 +29,5 @@ def execute(self):
         )
 
     def validate(self):
-        self.assertGrep("tedge.out", "tedge 0.5", contains=True)
+        self.assertGrep("tedge.out", "tedge 0.6", contains=True)
```

Comment:
Should this test manually updated on every minor update? I have to note it on the manual testing plan.

Reply:
Depends on how we like it. In the current state we only check the major version, which is kind of a compromise between having test coverage and adopting the test too often.
We can leave a note as pre release action to update versions where we have hardcoded ones.
We could also read the version from tedge.toml and make the test generic. However, in some small places I still like to have non generic checks as they will also detect typos in tedge.toml.